lib/cli/run.js

```diff
@@ -369,7 +369,12 @@ exports.handler = async function (argv) {
   try {
     await runMocha(mocha, argv);
   } catch (err) {
-    console.error('\n' + (err.stack || `Error: ${err.message || err}`));
+    if (!err) {
+      console.error('\n Undefined error:`, err);
+    } else {
+      console.error('\n' + (err.stack || `Error: ${err.message}`), err);
```

Comment:
Sometimes error does not have either stack or message. In that case it would be nice to `String(err)`

Reply:
in this approach you would see double message when a real error thrown